test/datasources.ts

```diff
@@ -806,3 +806,53 @@ export const mockRoleData = {
     },
   },
 };
+
+export const mockShowDatabasesPollingResult = {
```

Comment:
Might be worthwhile to type hint these with the intended types in case the type changes. Coming from [my recent PR](https://github.com/opensearch-project/dashboards-observability/pull/1498) where I paid the cost of not doing that sooner, and had to manually find/fix all the broken mocks.

Reply:
This is good point, updated the mocks!